src/styles/style-card.scss

```diff
@@ -0,0 +1,136 @@
+body {
+  margin: 0;
+  padding: 0;
+  --main-accent: #060b35;
+  --secondary: #616070;
```

Comment:
It's better to move variables to the :root selector to improve the code organization and reusability.